src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -50,16 +52,11 @@ protected function configure()
             ->setHelp(<<<'EOF'
 The <info>%command.name%</info> command dumps the current configuration for an
 extension/bundle.
-
```

Comment:
Can you revert all unrelated changes?

Reply:
I reverted all unrelated changes @fabpot.